client/me/purchases/remove-purchase/index.jsx

```diff
@@ -33,15 +33,15 @@ import notices from 'notices';
 import purchasePaths from '../paths';
 import { getPurchasesError } from 'state/purchases/selectors';
 import { removePurchase } from 'state/purchases/actions';
-import { isHappychatAvailable, hasActiveHappychatSession } from 'state/happychat/selectors';
+import { isHappychatAvailable, hasActiveHappychatSession } from 'extensions/happychat/state/selectors';
```

Comment:
Same here, if we aren't able to remove the direct references to the extension, we should at least comment prominently that it needs to be resolved.